app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.scss

```diff
@@ -24,8 +24,8 @@
 }
 
 .checkbox {
-  margin-bottom: 8px;
-  margin-top: 16px;
+  margin-bottom: 16px;
+  margin-top: 8px;
```

Comment:
It's still doesn't match design. you should remove .checkbox class from Checkbox component and managed everything via classNames property in FieldElement component